rules-tests/DeadCode/Rector/ClassMethod/RemoveUselessReturnTagRector/Fixture/remove_null.php.inc

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\ClassMethod\RemoveUselessReturnTagRector\Fixture;
+
+class RemoveNull
+{
+    /**
+     * @return null
+     */
+    function foo()
+    {
+        return null;
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\ClassMethod\RemoveUselessReturnTagRector\Fixture;
+
+class RemoveNull
+{
+    function foo()
```

Comment:
It seems cause no return tag specified phpstan notice, see https://phpstan.org/r/3c7579fb-dd25-4b55-94a3-637f15380990

Using `@return null` handle it, see https://phpstan.org/r/55a5277d-553c-4f7e-a9d7-840c9fd206f7

Reply:
Exactly, that's false positive that makes error hidden.

Reply:
I will create separate rule, it currenlty remove `@return null with description` as it invalid check